src/main/java/controllers/HouseLayoutController.java

```diff
@@ -61,28 +65,30 @@ public String readFromFile(String url){
         }
         return null;
     }
+
     /**
-     * help to extract all the object by converting the json object to RoomModel object
-     * @param jsonText
-     * @return
+     * This method will extract the information from the json and place it in the room model array.
+     * @param jsonText string to be read
+     * @return array from room model that contains all the rooms in the house layout file.
+>>>>>>> 761a4d6e1e469fffeaaaa709802c6f8b86cde5fe
```

Comment:
This is from a merge conflict! I think it needs to be deleted.

Reply:
ok, i delete already